src/object.js

```diff
@@ -83,10 +85,9 @@ export const compareDeep = _.curry(
   (path, item, value) => _.get(path, item) === value
 )
 
-// Applies a map function at a specific path
-// e.g.: mapProp(double, 'a', {a:2, b:1}) -> {a:4, b:1}
-export const mapProp = _.curry((fn, key, obj) =>
-  _.set(key, fn(_.get(key, obj)), obj)
+//Depreacted in favor of _.update version from lodash
+export const mapProp = aspects.deprecate('mapProp', '1.46.0', '_.update')(
+  noCap.mapProp
```

Comment:
ah ok - yeah my bad was just looking at the reference u mentioned in the ticket. Does that mean line 21 in conversion.js should also ref reduceIndexed? Ill put the unit test back in place...

Reply:
Not sure I follow what you mean about reducedIndexed - line 21 of conversion.js is a mutable set:

https://github.com/smartprocure/futil-js/blob/master/src/conversion.js#L21

I think with the unit test back in place we should be good.

Reply:
line 21 of conversion.js is using the noCap.reduce method, but from what i understood with what u mentioned to me before noCap.update being the method to replace the depreacted one.

Reply:
sorry... line 31 not 21 :D

Reply:
Got it - it could, but it's probably fine. All of the deprecated methods will be removed anyway, so as long as they're working I don't think there's any benefit in giving them much attention.